xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/Translations.fr.xml

```diff
@@ -127,11 +127,11 @@ notifications.menu.cleared=Les notifications ont été effacées
 notifications.settings.title=Préférences de notifications
 notifications.settings.error.notAdmin=Vous devez être administrateur pour visualiser cet élément.
 notifications.settings.menu=Menu des notifications
-notifications.settings.menu.hint=Le menu des notifications est situé en haut à droite de la page, vous pouvez l''ouvrir en cliquant sur la cloche ({0}).
+notifications.settings.menu.hint=Le menu des notifications est situé en haut à droite de la page, vous pouvez l''ouvrir en cliquant sur la cloche ({0})
 notifications.settings.email=Courriel
 notifications.settings.email.forGuest=Connectez-vous ou inscrivez-vous pour gérer vos paramètres de notification.
 notifications.settings.email.disabled=Les notifications par courriel sont désactivées. Veuillez contacter votre administrateur.
-notifications.settings.email.hint=Un courriel vous est envoyé régulièrement avec une synthèse de l'activité du wiki.
+notifications.settings.email.hint=Un mail vous est envoyé régulièrement avec une synthèse de l'activité du wiki
```

Comment:
I definitely use "mail" more than "courriel" myself too, but that's nonetheless the official word for it, so not sure it's right to change that.

Reply:
We should mostly check what word we use elsewhere in our translations to be consistent.

Reply:
Well, my feeling is that we should use "courriel" everywhere, "mail" is simply not a French word...

Reply:
it's a complex topic. A lot of people will cringe if you use "courriel" instead of "mail" (except in French-speaking Canada). Also there's the abbreviation `mél` (see https://www.larousse.fr/dictionnaires/francais/M%C3%A9l/50259 and https://fr.wiktionary.org/wiki/m%C3%A9l ) and `e-mail` (https://fr.wiktionary.org/wiki/e-mail#fr). Officially only `courriel` and `mél` are recognized by the French Academy but I'm not sure that it's a good criteria for what to use. I'd prefer to go with what is most natural for people to use (i.e what is the most used).

Reply:
We should definitely use `courriel` for `fr-ca` :)